components/molecules/Greeting.js

```diff
@@ -3,9 +3,11 @@ import { Message } from '@dts-stn/decd-design-system'
 export default function Greeting(props) {
   return (
     <div>
-      <div className="grid grid-cols-1 divide-y divide-red-600 font-display">
-        <div className="py-4 text-4xl font-bold">{`${props.greeting} ${props.name}`}</div>
-        <div className="py-4 text-xl">{props.welcome}</div>
+      <div className="grid grWid-cols-1 divide-y divide-red-600 font-display">
+        <h1 className="py-4 text-3xl font-bold text-gray-darker">
```

Comment:
We can actually be using the Heading component from the design system for our H1s. See the profile page
https://github.com/DTS-STN/sc-digital-centre/blob/fa7e1e9ecb9020d9930623fb713ed62b4bf9e5a9/pages/profile.js#L121